### DISCUSSION TITLE
Add GDScript reference to the Editor

### DISCUSSION THREAD

**Discussion by miv391** [godotengine/godot-proposals#11396] [Editor]
By GDScript reference I mean the documentation about how the basic building blocks of the language work. The reference is already in Internet: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html and it is very good.

However, it would be nice to have the reference in the Editor itself as a part of the documentation. Even an experienced programmer sometimes need answers to questions like this:
- What is the syntax of `enum`? 
- How does `static` keyword work?
- How to call function from a base class?

So it would be great to find the answers by clicking Ctrl + LMB as easily as you can get documentation about `print` function. The keywords (`if,` `else,` ...) should also be present in the Help (opened by F1).

Also when working offline having the GDScript reference built in would be nice.

**Comment by Mickeon** [comment]
There's already a proposal for this, and a corresponding PR:

- https://github.com/godotengine/godot-proposals/issues/2361
- https://github.com/godotengine/godot/pull/97196
